config/boot.rb

```diff
@@ -1,4 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+#require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
```

Comment:
I'm not familiar with this aspect of the config -- curious about why you chose to comment it out (and why you changed `rails` to `rails/all` in `config/application.rb`)